shared/src/models/vehicle-template.ts

```diff
@@ -20,10 +22,24 @@ export class VehicleTemplate {
     @IsString()
     public readonly name: string;
 
+    /**
+     * If an array, at position 0 it means zero patients are loaded
+     * at position 1 one patient is loaded, etc.
+     */
     @ValidateNested()
     @Type(() => ImageProperties)
     public readonly image: ImageProperties;
 
+    /**
+     * If an array, at position 0 it means zero patients are loaded
+     * at position 1 one patient is loaded, etc.
+     */
+    @IsArray()
+    @ValidateNested()
+    @Type(() => ImageProperties)
+    @IsOptional()
+    public readonly imagesPatientsLoaded?: ImageProperties[];
```

Comment:
Prefer `null` over `undefined` for new code (@Dassderdie I think we've come to that conclusion?):

```suggestion
    public readonly imagesPatientsLoaded: ImageProperties[] | null;
```

The validators should work the same, I've not tested it, though.